tests/src/Kernel/Cache/Context/OgRoleCacheContextTest.php

```diff
@@ -0,0 +1,248 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\og\Kernel\Cache\Context;
+
+use Drupal\Core\Cache\Context\CacheContextInterface;
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\Tests\og\Traits\OgRoleCacheContextTestTrait;
+use Drupal\Tests\user\Traits\UserCreationTrait;
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\node\Entity\Node;
+use Drupal\node\Entity\NodeType;
+use Drupal\og\Cache\Context\OgRoleCacheContext;
+use Drupal\og\Entity\OgMembership;
+use Drupal\og\Entity\OgRole;
+use Drupal\user\Entity\User;
+
+/**
+ * Tests the OG role cache context.
+ *
+ * @group og
+ * @coversDefaultClass \Drupal\og\Cache\Context\OgRoleCacheContext
+ */
+class OgRoleCacheContextTest extends KernelTestBase {
+
+  use OgRoleCacheContextTestTrait;
+  use UserCreationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'entity_test',
+    'node',
+    'og',
+    'system',
+    'user',
+    'field',
+  ];
+
+  /**
+   * The database connection.
+   *
+   * @var \Drupal\Core\Database\Connection
+   */
+  protected $database;
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * The group type manager.
```

Comment:
```suggestion
   * The OG group type manager.
```
Just like "The OG membership manager service"